backend/src/views/all_collection/map.js

```diff
@@ -0,0 +1,5 @@
+function (doc) {
+    if ( doc.links.filter(e => e.verb === 'includes').length >  0 ){
+      emit(doc.dc_title, doc);
```

Comment:
Never emit the whole document as a value. Use `include_docs` at query time instead.